src/tests/system/tests/test_ldap.py

```diff
@@ -50,3 +50,47 @@ def test_ldap__change_password(client: Client, ldap: LDAP, modify_mode: str):
 
     assert client.auth.ssh.password(user, new_pass), "Authentication with new correct password failed"
     assert not client.auth.ssh.password(user, old_pass), "Authentication with old incorrect password did not fail"
+
+
+@pytest.mark.ticket(bz=[1067476, 1065534])
+@pytest.mark.topology(KnownTopology.LDAP)
+def test_ldap__user_with_whitespace(client: Client, ldap: LDAP):
+    """
+    :title: user with a whitespace at beginning is able to login and "id"
+    :setup:
+        1. Add " space1" user with uid and password set to SSSD
```

Comment:
What do you mean by `set to SSSD`?

Reply:
Same for the next line

Reply:
It is a mistake. There should be `Add user to LDAP`